GOVERNANCE.md

```diff
@@ -90,3 +90,9 @@ in this project before it became open source:
 * Démian Janssen
 * Peter Salemink
 * Tony Xiang
+
+## PyPI project management
+
+Our Python projects are published on PyPI. On PyPI we have the following roles:
+- Owner: (part of) the TSC is owner of the project
+- Maintainer: the maintainers are also maintainer of PyPI projects
```

Comment:
This refers to PyPI but we have similar managed communities like Zenodo (citations) and conda-forge. Doesn't need to be done now, but let's not forget about it

Reply:
out of scope for now